bcl2fastq/index_summary.py

```diff
@@ -174,16 +258,16 @@ def update_pie_chart(run_alias, known_json, unknown_json):
     fraction = (known_count + pruned_count) / total_clusters * 100
 
     return {
-        'data': [{
-            'labels': ['Known', 'Unknown'],
-            'values': [known_count, pruned_count],
-            'type': 'pie',
-            'marker': {'colors': ['#349600', '#ef963b']},
-        }],
-        'layout': {
-            'title': 'Flow Cell Composition of Known/Unknown Indices'
-        }
-    }, 'Predicted clusters / produced clusters: {}%'.format(
+               'data': [{
+                   'labels': ['Known', 'Unknown'],
+                   'values': [known_count, pruned_count],
+                   'type': 'pie',
+                   'marker': {'colors': ['#349600', '#ef963b']},
+               }],
+               'layout': {
+                   'title': 'Flow Cell Composition of Known/Unknown Indices'
+               }
+           }, 'Predicted clusters / produced clusters: {}%'.format(
```

Comment:
I think this is over-indented. Did the PEP8 autoformatter do this?

Reply:
yes, I used the pycharm formatter